doc/attributes.md

```diff
@@ -11,12 +11,12 @@ The YaST UI for creating and editing a user offers the following attributes:
 | full name | gecos | | | |
 | name | name | | | |
 | password | password | | | |
-| system mail | | | | |
+| system mail | receive_system_mail | | | See `MailAliases` module |
 | disable login | * password starting by `!` | | | |
 | uid | uid | | | |
 | home dir | home | | | | |
-| home dir permission | | | | |
-| empty home | | yes | | |
+| home dir permission | umask | yes | | `useradd -K UMASK=666` |
+| empty home | | yes | | `rm -rf` after creating. No way to ignore `/usr/etc/skel` |
```

Comment:
Maybe `rm -rf` is not enough. See below screenshot

![Screenshot from 2021-09-16 16-18-33](https://user-images.githubusercontent.com/1691872/133639361-daf3fe57-eea1-4469-a0e9-43ea03a7f2da.png)


And also https://unix.stackexchange.com/questions/77127/rm-rf-all-files-and-all-hidden-files-without-error/77196

Reply:
A possible Ruby way, 

```
require 'pathname'

Pathname.new("/home/rmtesting").children.each { |f| f.rm_rf }
```